### ISSUE TITLE
Is any version compatible with react 15?

### ISSUE THREAD

**Issue by avi-raj1** [microsoft/applicationinsights-react-js#25]
My application run react 15 and I tried to move it to react 16. Used react-codemod but getting errors related to internal dependencies using React.Proptypes.

 Is there any version of application insights that can work with react 15?

I have sucessfully sent logs from a demo application to my app insights account using react 16. But when I downgraded version to react 15, It threw similar errors.

TL;DR: I need [applicationinsights-react-js](https://github.com/microsoft/applicationinsights-react-js) to work with react 15

**Comment by MSNev** [comment]
I went back through the history of the extension on the [reactjs-legacy branch of ApplicationInsights](https://github.com/Microsoft/ApplicationInsights-JS/tree/reactjs-legacy) and it appears that when this extension was created (around 4 years ago) it was targeted at 16.8.6.

The underlying SDK (including the latest), should work with the older version, but it looks like you would need to create your own version / functionality that is provided by this extension.
